backend/cmd/main.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"database/sql"
+	"fmt"
+
+	_storeDelivery "github.com/PUArallelepiped/PUN-street-Universal-Access/store/delivery"
+	_storeRepo "github.com/PUArallelepiped/PUN-street-Universal-Access/store/repository"
+	_storeUsecase "github.com/PUArallelepiped/PUN-street-Universal-Access/store/usecase"
+
+	"github.com/gin-gonic/gin"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
+
+	_ "github.com/lib/pq"
```

Comment:
what is this 
i no see where use it

Reply:
https://blog.wu-boy.com/2018/04/init-func-in-golang/